finance_helpers.py

```diff
@@ -60,17 +70,24 @@ def average_log_return(returns):
     """
     return np.log(np.mean(returns))
 
+
 def log_returns(returns):
     return np.log(returns)
 
+
 def volatility_of_returns(returns):
     """return volatility of returns"""
     lg_rets = log_returns(returns)
     return np.std(lg_rets)
 
-RISK_FREE_RATE = 0.0221 * (0.65) # tax (ltcg) adjusted
-def sharpe_ratio(returns, risk_free_rate = RISK_FREE_RATE):
-    """calculate (average) sharpe ratio.
+
+RISK_FREE_RATE = 0.0221 * (0.65)  # tax (ltcg) adjusted
+
+
+def sharpe_ratio(returns, risk_free_rate=RISK_FREE_RATE):
+    """
+    calculate (average) sharpe ratio.
     risk free rate uses 30 day t-bill ratio
     """
-    return (average_log_return(returns) - risk_free_rate)/volatility_of_returns(returns)
\ No newline at end of file
+    alpha = average_log_return(returns) - risk_free_rate
```

Comment:
I'm always confused about definitions here.  I thought "alpha" was the excess return relative to an index of equivalent beta, not relative to risk free rate? Or are there different "alphas"?

Reply:
Its context-specific.  The usage here is a close-enough because there isn't anything more appropriate.